samples/bluetooth/le_periph_pm/src/main.c

```diff
@@ -824,10 +1204,11 @@ int main(void)
 			}
 		}
 	}
-
-	if (IS_ENABLED(CONFIG_SLEEP_ENABLED)) {
-		power_mgr_ready_for_sleep();
-	}
+	app_pm_lock_deeper_states(false);
+	ret = app_enter_deep_sleep(RTC_CONNECTED_WAKEUP_INTERVAL_MS*1000);
```

Comment:
```suggestion
	ret = app_enter_deep_sleep(RTC_CONNECTED_WAKEUP_INTERVAL_MS * 1000);
```
Please run `checkpatch`!